Dockerfile.updater-core

```
@@ -0,0 +1,187 @@
+ && rm -rf /var/lib/gems/*/cache/* \
+ && rm -rf /tmp/ruby-install
+
+ENV HOME="/home/dependabot"
+WORKDIR ${HOME}
+
+# Place a git shim ahead of git on the path to rewrite git arguments to use HTTPS.
+ARG SHIM="https://github.com/dependabot/git-shim/releases/download/v1.4.0/git-v1.4.0-linux-amd64.tar.gz"
+RUN curl -sL $SHIM -o git-shim.tar.gz && mkdir -p ~/bin && tar -xvf git-shim.tar.gz -C ~/bin && rm git-shim.tar.gz
+
+# Disable automatic pulling of files stored with Git LFS
+# This avoids downloading large files not necessary for the dependabot scripts
+ENV GIT_LFS_SKIP_SMUDGE=1
+
+###############################################################################################
+# BEGIN UPDATER PASTE
+###############################################################################################
+
+ENV DEPENDABOT_HOME /home/dependabot
+
+RUN mkdir $DEPENDABOT_HOME/dependabot-updater
+
+COPY --chown=dependabot:dependabot updater/Gemfile updater/Gemfile.lock $DEPENDABOT_HOME/dependabot-updater/
+
+COPY --chown=dependabot:dependabot .ruby-version ${DEPENDABOT_HOME}/.ruby-version
+COPY --chown=dependabot:dependabot .rubocop.yml ${DEPENDABOT_HOME}/.rubocop.yml
+
+WORKDIR ${DEPENDABOT_HOME}
+COPY --chown=dependabot:dependabot omnibus ${DEPENDABOT_HOME}/omnibus
+COPY --chown=dependabot:dependabot git_submodules ${DEPENDABOT_HOME}/git_submodules
+COPY --chown=dependabot:dependabot terraform ${DEPENDABOT_HOME}/terraform
+COPY --chown=dependabot:dependabot github_actions ${DEPENDABOT_HOME}/github_actions
+COPY --chown=dependabot:dependabot hex ${DEPENDABOT_HOME}/hex
+COPY --chown=dependabot:dependabot elm ${DEPENDABOT_HOME}/elm
+COPY --chown=dependabot:dependabot docker ${DEPENDABOT_HOME}/docker
+COPY --chown=dependabot:dependabot nuget ${DEPENDABOT_HOME}/nuget
+COPY --chown=dependabot:dependabot maven ${DEPENDABOT_HOME}/maven
+COPY --chown=dependabot:dependabot gradle ${DEPENDABOT_HOME}/gradle
+COPY --chown=dependabot:dependabot cargo ${DEPENDABOT_HOME}/cargo
+COPY --chown=dependabot:dependabot composer ${DEPENDABOT_HOME}/composer
+COPY --chown=dependabot:dependabot go_modules ${DEPENDABOT_HOME}/go_modules
+COPY --chown=dependabot:dependabot python ${DEPENDABOT_HOME}/python
+COPY --chown=dependabot:dependabot pub ${DEPENDABOT_HOME}/pub
+COPY --chown=dependabot:dependabot npm_and_yarn ${DEPENDABOT_HOME}/npm_and_yarn
+COPY --chown=dependabot:dependabot bundler ${DEPENDABOT_HOME}/bundler
+COPY --chown=dependabot:dependabot common ${DEPENDABOT_HOME}/common
+
+WORKDIR $DEPENDABOT_HOME/dependabot-updater
+
+RUN bundle config set --local path 'vendor' && \
+bundle config set --local frozen 'true' && \
+bundle config set --local without 'development' && \
+bundle install
+
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND="noninteractive" \
+  LC_ALL="en_US.UTF-8" \
+  LANG="en_US.UTF-8"
+
+RUN apt-get update \
+  && apt-get upgrade -y \
+  && apt-get install -y --no-install-recommends \
+    # dev dependencies for CI
+    zlib1g-dev \
```

Oops forgot this one too!

```suggestion
    build-essential \
    zlib1g-dev \
```